src/components/StorageOverview/Utilization/tests/Utilization.test.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import { render } from 'enzyme';
+import { shallow } from 'enzyme';
 
 import { Utilization } from '../Utilization';
 import { default as UtilizationFixtures } from '../fixtures/Utilization.fixture';
 
 describe('<Utilization />', () => {
   it('renders correctly', () => {
-    const component = render(<Utilization {...UtilizationFixtures.props} />);
+    const component = shallow(<Utilization {...UtilizationFixtures.props} />);
```

Comment:
I am fine with this change but can you briefly explain reasons for it, please?

Reply:
If we use `render`, it will render everything, including the child components resulting in a huge snapshot which will not be useful. Using `shallow` , only the single component is rendered and we get a smaller and more useful snapshots.